src/main/java/com/powsybl/caseserver/elasticsearch/ESConfig.java

```diff
@@ -30,40 +26,10 @@
 
 @Configuration
 @EnableElasticsearchRepositories
```

Comment:
Do we still need @EnableElasticsearchRepositories ?

Reply:
you're right, I've removed it and made some tests, it's still working without it